src/gui/advancedsettings.cpp

```diff
@@ -418,7 +421,10 @@ void AdvancedSettings::loadAdvancedSettings()
     // Download tracker's favicon
     cb_tracker_favicon.setChecked(mainWindow->isDownloadTrackerFavicon());
     addRow(DOWNLOAD_TRACKER_FAVICON, tr("Download tracker's favicon"), &cb_tracker_favicon);
-
+    // Save path history length
+    spinDownloadPathHistoryLength.setRange(0, 100);
```

Comment:
http://doc.qt.io/qt-5/qspinbox.html#maximum-prop
>The default maximum value is 99.

So maybe you would like to omit this.

Reply:
Who want to remember 100 download paths? I think it should be set up to something meaningful, like 30.

Reply:
@Chocobo1: thanks, omitted.
@silverqx: I don't know, current 8 is more than enough for me (typing is faster than searching in longer lists), but people ask. So I don't want to limit them.